packages/xchain-bitcoin/src/client.ts

```diff
@@ -162,84 +150,11 @@ class Client implements BitcoinClient, XChainClient {
   }
 
   // Will return true/false
-  validateAddress = (address: string): boolean => {
-    const btcNetwork = this.btcNetwork()
-    try {
-      Bitcoin.address.toOutputScript(address, btcNetwork)
-      return true
-    } catch (error) {
-      return false
-    }
-  }
-
-  // Scans UTXOs on Address
-  scanUTXOs = async (address?: Address): Promise<void> => {
-    try {
-      if (!address) {
-        address = this.getAddress()
-      }
-      const dashboardsAddress = await blockChair.getAddress(this.nodeUrl, address, this.nodeApiKey)
-
-      this.utxos = [] // clear existing utxos
-      const utxos = dashboardsAddress[address].utxo
-
-      for (let i = 0; i < utxos.length; i++) {
-        const txHash = utxos[i].transaction_hash
-        const value = utxos[i].value
-        const index = utxos[i].index
-        const txData = await blockChair.getRawTx(this.nodeUrl, txHash, this.nodeApiKey)
-        const script = txData[txHash].decoded_raw_transaction.vout[index].scriptPubKey.hex
-        // TODO: check scriptpubkey_type is op_return
-
-        const witness = {
-          value: value,
-          script: Buffer.from(script, 'hex'),
-        }
-
-        const utxoObject = {
-          hash: txHash,
-          index: index,
-          witnessUtxo: witness,
-          txHex: txData[txHash].raw_transaction,
-        }
-        this.utxos.push(utxoObject)
-      }
-    } catch (error) {
-      throw new Error(error)
-    }
-  }
+  validateAddress = (address: string): boolean => Utils.validateAddress(address, this.net)
 
   // Returns balance of address
-  getBalance = async (address?: string): Promise<Balance[]> => {
-    if (!address) {
-      address = this.getAddress()
-    }
-
-    try {
-      // const chain = this.net === 'testnet' ? 'bitcoin/testnet' : 'bitcoin'
-      const dashboardAddress = await blockChair.getAddress(this.nodeUrl, address, this.nodeApiKey)
-      return [
-        {
-          asset: AssetBTC,
-          amount: baseAmount(dashboardAddress[address].address.balance),
-        },
-      ]
-    } catch (error) {
-      return Promise.reject(new Error('Invalid address'))
-    }
-  }
-
-  // Given a desired output, return change
-  private getChange = async (valueOut: number, address?: string): Promise<number> => {
-    const balances = await this.getBalance(address)
-    const btcBalance = balances.find((balance) => assetToString(balance.asset) === assetToString(AssetBTC))
-    let change = 0
-
-    if (btcBalance && btcBalance.amount.amount().minus(valueOut).isGreaterThan(Utils.dustThreshold)) {
-      change = btcBalance.amount.amount().minus(valueOut).toNumber()
-    }
-    return change
-  }
+  getBalance = async (address?: string): Promise<Balance[]> =>
+    Utils.getBalance(address || this.getAddress(), this.nodeUrl, this.nodeApiKey)
 
   // Get transaction for the address
   getTransactions = async (params?: TxHistoryParams): Promise<TxsPage> => {
```

Comment:
Same[ in next line](https://github.com/xchainjs/xchainjs-lib/pull/135/files#diff-445eb8a0f4b2e25d808c6145e18e3bd7945a0e6b56424357f7d3c494007fa732R161) (_I'm unable to add a comment there_): this.getAddress() might fail and we should return Promise.reject(...) then.